usr/lib/api/api_interface.c

```diff
@@ -4864,21 +4864,44 @@ CK_RV C_LoginUser(CK_SESSION_HANDLE hSession, CK_USER_TYPE userType,
 CK_RV C_SessionCancel(CK_SESSION_HANDLE hSession, CK_FLAGS flags)
 {
     CK_RV rv;
-
-    UNUSED(hSession);
-    UNUSED(flags);
+    API_Slot_t *sltp;
+    STDLL_FcnList_t *fcn;
```

Comment:
Are there spaces for indentation in line 4867?

Reply:
Yes there are spaces. Github does not color the spaces because the were already there with the removed line content.